schema/album_and_genre.sql

```diff
@@ -0,0 +1,22 @@
+CREATE DATABASE catalog_of_things;
+
+CREATE TABLE MusicAlbum(
+  id            INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  genre_id      INTEGER,
+  author_id     INTEGER,
+  label_id      INTEGER,
```

Comment:
Hi @see-why, do we have a datatype in PostgreSQL called "INTEGER"? Do you mean "INT"? Please, let's check the correct data type well before merging this pull request. Thank you.

Reply:
Thanks a lot for the feedback, it turns out that we are both correct
INT and INTEGER can be used interchangeably.

The next time I need to write a schema I will use INT since it is shorter in length.

Thanks again.

Reply:
Oh! Great to learn this. Thank you for your professional response. I will approve your request accordingly. 
:+1: